### ISSUE TITLE
Error: Uh! oh! Something went wrong We appreciate your patience. While we fix this, continue browsing on the app

### ISSUE THREAD

**Issue by ajithrameshcr** [razorpay/razorpay-flutter#408]
### This error occasionally appears after making a UPI payment. Although the account has already received payment, there are no Success or Failure notifications displayed.
![WhatsApp Image 2024-08-24 at 13 47 03](https://github.com/user-attachments/assets/6edb647e-29a2-45e0-b11f-feca2741a54b)

**Comment by SuryaNarayananDev** [comment]
Are you cleared ?

**Comment by ajithrameshcr** [comment]
Not exactly, we are getting an issue (please check the attachment) while
doing UPI payment in both Test and Live Mode in some devices. Money is
already captured every time even it is failed by the so and so error. We
need to know when it is happen and Why.


~Regards
Ajith Ramesh CR
Mobile App developer

**Comment by SuryaNarayananDev** [comment]
how rectifie this prob?

**Comment by irudra** [comment]
Do we have any update on this issue? We are also facing similar issue in test mode?

**Comment by SuryaNarayananDev** [comment]
> Do we have any update on this issue? We are also facing similar issue in test mode?

First I Think Code is problem then , the real problem is on API key , S-Key og Api
If U face this problem 
    1.go to razorpay dashborad then regenerate key again 
    2.take new key then enter in your code then problem will sortout 
if face any problem after above steps sent repo link check

**Comment by irudra** [comment]
Thank you, It turned out we were using different key in the release environment.

**Comment by brandwavestudio** [comment]
"currency": "INR",
"name": "Astroo Guru",
"description": "Connect with experienced astrologers close to you.",
 "image": "god.png",
//"account_id": "acc_Ef7ArAsdU5t0XL",
//"order_id": "order_DBJOWzybf0sJbb",

This is the Razorpay code which we have to pass dynamically.  If you comment order_id your error will be solve but you have to pass this dynamicaaly otherwise your client will get his paid payment  back after few days.

[
For high-quality ecommerce projects, you can contact me. ](https://www.brandwavestudio.com )

**Comment by GokupcWork** [comment]
any update still facing the same error from weeks ..

**Comment by Avinash-Tallapaneni** [comment]
Anyone fixed this issue?


```
"react": "19.0.0",
"react-native": "0.79.2",
"react-native-razorpay": "^2.3.0"
```
```
import RazorpayCheckout from 'react-native-razorpay';

const options = {
      description: `${totalItems} item(s)`,
      image: 'https://i.imgur.com/3g7nmJC.png',
      currency: 'INR',
      key: 'rzp_test_kEY',
      amount: 400,
      name: 'SimpleCart',
      order_id: 'order_sample123',
      prefill: {
        email: 'test@example.com',
        contact: '911234567890',
        name: 'Customer',
      },
      theme: {color: '#6366f1'},
};

    RazorpayCheckout.open(options)
      .then(data => {
        console.log('Payment success:', data);

        Alert.alert('Payment Success', `ID: ${data.razorpay_payment_id}`);
        setCart({});
      })
      .catch(error => {
        console.log('Payment Error:', error);
        Alert.alert('Payment Failed', `${error.code} | ${error.description}`);
      });
```

![Image](https://github.com/user-attachments/assets/5dbc26a9-d659-4b6f-ac15-e1b076f6ed9b)

**Comment by JadhavC07** [comment]
Payment is successful but got error "Something went Wrong , please try again" In flutter

**Comment by SuryaNarayananDev** [comment]
if show this error , go again create new razorpay api key the problem gone!

**Comment by JadhavC07** [comment]
> if show this error , go again create new razorpay api key the problem gone!

OKay , I will try

**Comment by SuryaNarayananDev** [comment]
if again prone the error share the error phase or keyword like scr-shot , web browser console

**Comment by venkateshkasani** [comment]
![Image](https://github.com/user-attachments/assets/a766c500-fb5a-4585-beb7-b649ed260481)
I am getting this on my nextjs app and it is not even making any test transactions at all.
Here is the trigger function code :

` const handleClick = async (e) => {
        e.preventDefault();
        const res = await fetch('http://localhost:3000/payments/createPayment', {
             headers: {
      "Content-Type": "application/json", 
    },
            method: "POST",
            body:JSON.stringify({"amount":1000,"itinerary":"0"})
        })
        const data = await res.json();
        const paymentData = {
            ...data,
            key: process.env.RAZORPAY_API_KEYID,
            handler: async function (response: any) {
                // verify
            },
              prefill: {
        "name": "Gaurav Kumar", 
        "email": "gaurav.kumar@example.com", 
        "contact": "+919876543210"
    },
    notes: {
        "address": "Razorpay Corporate Office"
    },
    theme: {
        "color": "#3399cc"
    }
        }
        const payment = new (window as any).Razorpay(paymentData)
        payment.open();
    }
`

**Comment by mjjabarullah** [comment]
Same issue here

**Comment by SuryaNarayananDev** [comment]
@mjjabarullah  when , after paymeny or initialize( invoke payment ) ?

**Comment by Arunkashyap29** [comment]
<img width="1248" height="707" alt="Image" src="https://github.com/user-attachments/assets/e2576adc-43db-4749-ac94-d988b9949c45" />
I have a WordPress website where I have integrated Razorpay. The payment is successful, the order is placed, and the confirmation email has been sent. However, I still get an error on the thank‑you page: “Uh! oh! Something went wrong.”
